packages/common/src/components/Preview/Navigator/elements.ts

```diff
@@ -0,0 +1,86 @@
+import styled, { css } from 'styled-components';
+import Color from 'color';
+
+export const darker = (light: boolean, color: string) =>
+  Color(color)
+    .darken(light ? 0.3 : 0.7)
+    .hexString();
+
+export const Container = styled.div`
+  display: flex;
+  background-color: ${props =>
+    props.theme['input.background'] || props.theme.background4};
+  padding: 0.25rem;
+  align-items: center;
+  line-height: 1;
+  /* box-shadow: 0 1px 3px #ddd; */
+  height: 35px;
+  min-height: 35px;
+  box-sizing: border-box;
+  z-index: 2;
+`;
+
+export const Icons = styled.div`
+  display: flex;
+`;
+
+export const Icon = styled.button<{ bg?: boolean; moduleView?: boolean }>`
+  display: inline-block;
+  border: none;
+  background-color: transparent;
+  font-size: 1.5rem;
+  line-height: 0.5;
+  margin: 0 0.25rem;
+  vertical-align: middle;
+  text-align: center;
+  padding: 0;
+  outline: none;
+  cursor: pointer;
+
+${({ bg, theme }) =>
+  bg &&
```

Comment:
bg: boolean? what does this do?

Reply:
Its for these icons 

![Screenshot 2019-12-16 at 15 55 41](https://user-images.githubusercontent.com/1051509/70916808-8cb3fb00-201c-11ea-9e64-f6e683a22158.png)

Reply:
Its for thse icons
![Screenshot 2019-12-16 at 15 55 41](https://user-images.githubusercontent.com/1051509/70917301-5b87fa80-201d-11ea-9380-ba3e76015956.png)

Reply:
should we call it a more descriptive name. I couldn't have guessed

Reply:
Named it `IconWithBackground` and made a new styled component